package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "tdr-auth-server",
+  "version": "0.1.0",
+  "description": "Repository for TDR Auth Server code",
+  "private": true,
+  "scripts": {
+    "copy-govuk-image-assets": "copyfiles -f node_modules/govuk-frontend/govuk/assets/images/* ./themes/tdr/$npm_config_theme/resources/img -e node_modules/govuk-frontend/govuk/assets/images/favicon*",
+    "copy-govuk-font-assets": "copyfiles -f node_modules/govuk-frontend/govuk/assets/fonts/* ./themes/tdr/$npm_config_theme/resources/fonts",
+    "copy-govuk-js-assets": "copyfiles -f node_modules/govuk-frontend/govuk/all.js ./themes/tdr/$npm_config_theme/resources/js",
+    "copy-assets": "npm-run-all copy-govuk-image-assets copy-govuk-font-assets copy-govuk-js-assets",
+    "sass-compile": "node-sass ./themes/tdr/css-src/sass/main.scss ./themes/tdr/css-src/main.css",
+    "add-stylesheet-dir": "mkdir -p ./themes/tdr/$npm_config_theme/resources/css",
+    "compress-css": "minify ./themes/tdr/css-src/main.css > ./themes/tdr/$npm_config_theme/resources/css/main.css",
+    "build-css": "npm-run-all add-stylesheet-dir sass-compile compress-css",
+    "build-login": "npm-run-all copy-assets build-css --theme=login",
+    "update-local-resources": "docker cp themes/tdr/$npm_config_theme/. $npm_config_container_name:/opt/jboss/keycloak/themes/tdr/$npm_config_theme",
+    "build-local": "npm-run-all build-login update-local-resources",
+    "build-theme": "npm-run-all build-login"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nationalarchives/tdr-auth-server.git"
+  },
+  "author": "The National Archives",
+  "license": "MIT",
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "bugs": {
+    "url": "https://github.com/nationalarchives/tdr-auth-server/issues"
+  },
+  "homepage": "https://github.com/nationalarchives/tdr-auth-server#readme",
+  "dependencies": {
+    "govuk-frontend": "^3.9.1",
+    "unfetch": "^4.1.0"
+  },
+  "devDependencies": {
+    "copyfiles": "^2.4.0",
```

Comment:
That reminds me: we should add Dependabot to this repo. Example from the frontend: https://github.com/nationalarchives/tdr-transfer-frontend/blob/master/.github/dependabot.yml

Reply:
I will create a separate PR for this.